src/infra/src/table/entity/users.rs

```diff
@@ -0,0 +1,36 @@
+//! `SeaORM` Entity, @generated by sea-orm-codegen 1.1.0
+
+use sea_orm::entity::prelude::*;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "users")]
+pub struct Model {
+    #[sea_orm(primary_key, auto_increment = false)]
+    pub id: String,
+    #[sea_orm(unique)]
+    pub email: String,
+    pub first_name: String,
+    pub last_name: String,
+    #[sea_orm(column_type = "Text")]
+    pub password: String,
+    pub salt: String,
+    pub is_root: bool,
+    pub password_ext: Option<String>,
+    pub user_type: i16,
+    pub created_at: i64,
+    pub updated_at: i64,
+}
```

Comment:
Why password need `text`? and what is password_ext?

Reply:
I kept that as `TEXT` because I am not sure about the possible number of characters it can take. Regarding `password_ext`, I am not sure what is that for. It was there earlier, hence I kept it. @oasisk may help!